src/routes/console/project-[project]/auth/+layout.svelte

```diff
@@ -1,3 +1,131 @@
+<script lang="ts">
+    import { goto } from '$app/navigation';
+    import { page } from '$app/stores';
+    import { registerCommands, updateCommandGroupRanks } from '$lib/commandCenter';
+    import { project } from '../store';
+    import { showCreateUser } from './+page.svelte';
+    import { showCreateTeam } from './teams/+page.svelte';
+
+    $: $registerCommands([
+        {
+            label: 'Create user',
+            callback: async () => {
+                if (!$page.url.pathname.endsWith('auth')) {
+                    await goto(`/console/project-${$project.$id}/auth`);
+                }
+                showCreateUser.set(true);
+            },
+            keys: $page.url.pathname.endsWith('auth') ? ['c'] : ['c', 'u'],
+
+            group: 'users',
+            icon: 'plus',
+            rank: $page.url.pathname.endsWith('auth') ? 10 : 0
+        },
+        {
+            label: 'Create team',
+            callback: async () => {
+                if (!$page.url.pathname.endsWith('teams')) {
+                    await goto(`/console/project-${$project.$id}/auth/teams`);
+                }
+                showCreateTeam.set(true);
+            },
+            keys: $page.url.pathname.endsWith('teams') ? ['c'] : ['c', 't'],
+
+            group: 'teams',
+            icon: 'plus',
+            rank: $page.url.pathname.endsWith('teams') ? 10 : 0
+        },
+        {
+            label: 'Go to teams',
+            keys: ['g', 't'],
+            callback() {
+                goto(`/console/project-${$project.$id}/auth/teams`);
+            },
+            group: 'navigation',
+            rank: 1,
+            disabled: $page.url.pathname.endsWith('teams')
+        },
+        {
+            label: 'Go to usage',
+            keys: ['g', 'u'],
+            callback() {
+                goto(`/console/project-${$project.$id}/auth/usage`);
+            },
+            group: 'navigation',
+            rank: 1,
+            disabled: $page.url.pathname.endsWith('usage')
+        },
+        {
+            label: 'Go to security',
+            keys: ['g', 'e'],
+            callback() {
+                goto(`/console/project-${$project.$id}/auth/security`);
+            },
+            group: 'navigation',
+            rank: 1,
+            disabled: $page.url.pathname.endsWith('security')
+        },
+        {
+            label: 'Go to settings',
+            keys: ['g', 's'],
+            callback() {
```

Comment:
Why are some of the callback async and some not even if they all redirect somewhere else?

Reply:
I think this is solved now